rotkehlchen/db/upgrades/v27_v28.py

```diff
@@ -18,3 +18,4 @@ def upgrade_v27_to_v28(db: 'DBHandler') -> None:
             'ALTER TABLE yearn_vaults_events ADD COLUMN version INTEGER NOT NULL DEFAULT 1;',
         )
     db.conn.commit()
+    db.delete_aave_data()
```

Comment:
Add some mock aave data in the test DB and check they exist and they are deleted after the upgrade

Reply:
Isn't this testing again that the `delete_aave_data` works?

Reply:
no it's testing that you added the aave data in the upgrade. For example ... here you have added the deletion before the commit. You should move it above also.